src/serialport.cpp

```diff
@@ -681,6 +731,7 @@ extern "C" {
     Nan::SetMethod(target, "open", Open);
     Nan::SetMethod(target, "update", Update);
     Nan::SetMethod(target, "write", Write);
+    Nan::SetMethod(target, "read", Read);
```

Comment:
I'd like to leave it since serialport_uwp will also use it. I can use ifdef WIN32 since the macro is defined when UWP is used.